resources/css/register.css

```diff
@@ -13,3 +13,22 @@
     width: 100%;
     max-width: 200px;
 }
+
+.register__background {
+    --dt-tile-background-color: hsla(0, 0%, 100%, 1);
```

Comment:
Please change the name of magic-link.css to theme.css and set --dt-tile-background-color in theme.css, and remove line 17/18/19.